src/workflows-app/components/FilterableWorkflowTable.ts

```diff
@@ -310,6 +425,61 @@ const FilterableWorkflowTable = ({
                           }
                         },
                       },
+                      {
+                        size: { basis: 400, grow: 0 },
+                        field: 'taskData',
+                        headerRenderer: () => 'Workflow Data',
+                        cellRenderer: ({ rowIndex }) => {
+                          return h(TextCell, [
+                            h(
+                              Link,
+                              {
+                                onClick: async () => {
+                                  const workflow: WorkflowMetadata = (await loadWorkflows(
+                                    paginatedPreviousRuns[rowIndex].engine_id as string
+                                  )) as WorkflowMetadata;
```

Comment:
Are these casts necessary? Especially `paginatedPreviousRuns`... it is typed, so `engine_id` should also be typed.